### ISSUE TITLE
Recorded calls are open by Spotify

### ISSUE THREAD

**Issue by tu0mas** [chenxiaolong/BCR#281]
After a call, when I open the notification saying the call has been recorded, it will play the recorded call with Spotify exclusively.
I can't find where to modify this setting even though I have a player installed and able to play mp4 files.

Reinstalling both BCR and Spotify didn't work.

**Comment by chenxiaolong** [comment]
Hmm, I have no idea why this would be happening. BCR doesn't ask for a specific app. Android is supposed to show the usual popup for selecting an app (any one that can handle the `audio/mp4` file type).

Which audio player do you have installed?

**Comment by tu0mas** [comment]
Oh okay. I told myself this is rather a OS/Android issue but I wasn't sure. I tried switching from mp4 to ogg in BCR recording settings, Spotify will still open it. I have MX Player installed and it will play mp4 files by default if I open them via File Explorer which is weird.

**Comment by chenxiaolong** [comment]
I did some investigation into this. It looks like this is an MX Player bug unfortunately.

The app picker popup only shows up with MX Player if the original app reports that the file type is unknown (`audio/*`). This is the case with some file managers that cannot correctly detect the file type, eg. Solid Explorer.

When the file type is known (`audio/mp4` for m4a or `audio/ogg` for ogg), then MX Player doesn't show up. Aside from BCR's notification, I can reproduce this in other file managers that correctly detect the type too, like AOSP's built in file manager and Google Files.

Other music players don't seem to have this issue. (I tested VLC, mpv, Retro Music, and Youtube Music.)
